cmds/webboot/webboot.go

```diff
@@ -267,6 +278,11 @@ func handleError(err error) {
 	wifiStderr.Reset()
 }
 
+func showLog() {
+	menu.DisplayResult(strings.Split(logBuffer.String(), "\n"), ui.PollEvents())
```

Comment:
but you threw lots of output away above. I assume this is a WIP?

Reply:
Sort of. The extensive output causes it to get lost entirely, and from within the interactive menu, there is zero feedback otherwise, which is why I started this here in the first place. Yes, it's a WIP, and I think it would make sense to offer another CLI arg for a path to a log file. The menu is not suitable for long text. Especially because scrolling down leaves the scroll output, so it's lost.

Reply:
Or in other words: Feedback within the interactive menu should be short enough to navigate it, and extensive output (the "full" version) written elsewhere, like a file.

Reply:
Okay I have a simple idea now, thank you!